src/Directory.Build.props

```diff
@@ -37,6 +37,7 @@
     <AzureAISearchVersion>0.3.0-alpha</AzureAISearchVersion>
     <!-- Version for CosmosDB search webjobs and worker project -->
     <CosmosDBSearchVersion>0.2.0-alpha</CosmosDBSearchVersion>
+    <CosmosDBNoSqlSearchVersion>0.16.0-alpha</CosmosDBNoSqlSearchVersion>
```

Comment:
the versioning for this will start with 0.1.0-alpha

also add a changelog file for this as it is present for CosmosDB VCore

Search for `CosmosDBSearchVersion` in the entire repo for its usage and add the same for `CosmosDBNoSqlSearchVersion`